test/default-messages.js

```diff
@@ -1,37 +1,31 @@
 var tap = require('tap');
 var spawn = require('child_process').spawn;
-var trim = require('string.prototype.trim');
+var concat = require('concat-stream');
 
 tap.test('default messages', function (t) {
     t.plan(1);
 
-    var tc = tap.createConsumer();
+    var ps = spawn(process.execPath,
+        [ require('path').join(__dirname, 'messages', 'defaults.js') ]);
```

Comment:
The first example is exactly what I had in mind.

Personally I find "max line length" restrictions silly. If your first example is too long for you, you could do:
```js
var path = require('path');
var ps = spawn(
  process.execPath,
  [
    path.join(__dirname, 'messages', 'defaults.js')
  ]
);
```

(in either case, please move the require to a variable at the top level)